packages/mux-player/src/media-theme-mux.html

```diff
@@ -218,10 +214,10 @@
     media-loading-indicator {
       --media-loading-icon-width: 100%;
       --media-button-icon-height: auto;
+      display: var(--media-control-display, var(--media-loading-indicator-display, flex));
```

Comment:
question(blocking): Is this order right? Wouldn't we want the more specific `--media-loading-indicator-display` to take precedence over the more generic `--media-control-display`?